### PR TITLE
dtv: Fix gcc8 -Wclass-memaccess warnings/errors.

### PR DESCRIPTION
**Pull request by drmpeg**

### CODE REVIEW

**Comment by drmpeg** [comment]
I only felt comfortable changing the -Wclass-memaccess errors in the DTV component since I can do a bit perfect check of the changes. Remaining files with -Wclass-memaccess errors are:

gr-blocks/lib/mute_impl.cc
gr-filter/lib/fir_filter.cc
gr-filter/lib/fir_filter_with_buffer.cc
gr-filter/lib/pfb_synthesizer_ccf_impl.cc
gr-filter/lib/qa_fir_filter_with_buffer.cc
gr-qtgui/lib/time_sink_c_impl.cc
gr-qtgui/lib/freq_sink_c_impl.cc
gr-qtgui/lib/waterfall_sink_c_impl.cc
gr-digital/lib/burst_shaper_impl.cc

File: gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.cc [hunk 1]
```diff
@@ -751,8 +752,8 @@ namespace gr {
               papr_fft->execute();
               memcpy(ones_time, papr_fft->get_outbuf(), sizeof(gr_complex) * papr_fft_size);
               volk_32fc_s32fc_multiply_32fc(ones_time, ones_time, normalization, papr_fft_size);
-              memset(&r[0], 0, sizeof(gr_complex) * N_TR);
-              memset(&c[0], 0, sizeof(gr_complex) * papr_fft_size);
+              fill_n(&r[0], N_TR, 0);
```
Comment (mbr0wn) [inline gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.cc]: Are you `using namespace std;` or why does this work without the `std::`?

**Comment by mbr0wn** [comment]
This is a great change!

File: gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.cc (hunk 1 above)
Comment (drmpeg) [reply gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.cc]: Thanx for catching that. I definitely meant to use std::. I must of checked it 5 times, but still overlooked it.

As to why it still compiles, I'm not sure. I did notice that "using namespace std;" is included in all xxx_swigPYTHON_wrap.cxx swig files. For example, build/gr-dtv/swig/dtv_swigPYTHON_wrap.cxx

How it finds the function is also interesting.
```
In file included from /usr/include/c++/8/memory:62,
                 from /usr/include/boost/config/no_tr1/memory.hpp:21,
                 from /usr/include/boost/smart_ptr/shared_ptr.hpp:23,
                 from /usr/include/boost/shared_ptr.hpp:17,
                 from /home/re/xfer/gnuradio/gnuradio-runtime/include/gnuradio/sptr_magic.h:26,
                 from /home/re/xfer/gnuradio/gnuradio-runtime/include/gnuradio/basic_block.h:27,
                 from /home/re/xfer/gnuradio/gnuradio-runtime/include/gnuradio/block.h:28,
                 from /home/re/xfer/gnuradio/gnuradio-runtime/include/gnuradio/sync_block.h:27,
                 from /home/re/xfer/gnuradio/gr-dtv/include/gnuradio/dtv/dvbt2_paprtr_cc.h:27,
                 from /home/re/xfer/gnuradio/gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.h:24,
                 from /home/re/xfer/gnuradio/gr-dtv/lib/dvbt2/dvbt2_paprtr_cc_impl.cc:25:
/usr/include/c++/8/bits/stl_algobase.h:
```

I pushed a squashed commit with the fix.

**Pull request merged**